pkg/thing/delivery/http/thing_proxy.go

```diff
@@ -83,7 +83,7 @@ func NewThingProxy(logger logging.Logger, hostname string, port uint16) ThingPro
 // Create register a thing on service and return the id generated
 func (p proxy) Create(id, name, authorization string) (idGenerated string, err error) {
 	p.logger.Debug("proxying request to create thing")
-	t := p.getRemoteThingRepr(id, name, nil, nil)
+	t := p.getRemoteThingRepr(id, name, nil)
 	body, err := json.Marshal(t)
 	if err != nil {
```

Comment:
In this file, there is also a struct named `objKnot` that has an outdated `Schema` property. Remove it (the property) please 😉

Reply:
Done!